splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -131,6 +136,20 @@ private void configureHttpFailed() throws Exception {
                 .to(direct("return"));
     }
 
+    private void configureSecureConnectionFailed() throws Exception {
+        Processor ceEncoder = new CloudEventEncoder(COMPONENT_NAME, RETURN_TYPE);
+        Processor resultTransformer = new ResultTransformer();
+        // The error handler when we receive an HTTP (unsecure) connection instead of HTTPS
+        from(direct("secureConnectionFailed"))
+                .setBody(simple("Unsecure connection detected"))
+                .setHeader("outcome-fail", simple("true"))
+                .process(resultTransformer)
+                .marshal().json()
+            .log(LoggingLevel.ERROR, "Unsecure connection detected, id ${header.ce-id}")
+            .process(ceEncoder)
```

Comment:
I've noticed that this encoder process can be moved completely to "return" route.

Reply:
Moved it to `return` route.

Thanks